addons/api/addon/services/indexed-db.js

```diff
@@ -29,6 +29,8 @@ export const modelIndexes = {
     '&id, attributes.created_time, attributes.type, attributes.name, attributes.description, attributes.scope.scope_id, attributes.plugin.name',
   'auth-method':
     '&id, attributes.created_time, attributes.type, attributes.name, attributes.description, attributes.is_primary, attributes.scope.scope_id',
+  'session-recording':
+    '&id, attributes.created_time, attributes.type, attributes.state, attributes.start_time, attributes.end_time, attributes.duration, attributes.scope.scope_id, attributes.create_time_values.user.id, attributes.create_time_values.user.name, attributes.create_time_values.target.id, attributes.create_time_values.target.name, attributes.create_time_values.target.scope.id, attributes.create_time_values.target.scope.name',
```

Comment:
non-blocking: so basically for the scopes filter, we mainly using target scope because  we expect user scope id to be the same?

Reply:
I believe that is partially true. If we used the scope of the user that may not be helpful. Especially since an admin user's scope could be global. So we use the scope tied to the target that was used to generate the session recording. There was even a comment in the designs about only showing the projects in the filter but I used our standard scope filter of showing the org and what projects are part of that org since that is how we already display the scope filter in the desktop client.